### PR TITLE
Update README to add documentation about data sources.

### PR DESCRIPTION
**Pull request by oliverchang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,15 +17,22 @@ query whether or not their versions are impacted.
   <img src="docs/images/diagram.png" width="600">
 </p>
 
-Current data sources:
-- [OSS-Fuzz](https://github.com/google/oss-fuzz) (https://github.com/google/oss-fuzz-vulns)
+## Current data sources
+**This is an ongoing project.** We are hoping to work with the open source community
```
Comment (inferno-chromium) [inline README.md:21]: Can use slightly stronger tone. like "We encourage open source ecosystems to try our proposed vulnerability interchange schema format (link) and integrate with our existing database, for the benefit of the open source community. The following ecosystems have accepted our schema format and integrated with us:"

File: README.md [hunk 2]
```diff
@@ -17,15 +17,22 @@ query whether or not their versions are impacted.
...
 - [DWF](https://github.com/distributedweaknessfiling/dwflist)
 
-**This is an ongoing project.** We are hoping to work with the open source community
-to [onboard more sources of data](https://github.com/google/osv/issues/44).
+For convenience, these sources are aggregated and continuously exported to a GCS bucket
+maintained by OSV: [gs://osv-vulnerabilities](https://osv-vulnerabilities.storage.googleapis.com).
+
+This bucket contains individual entries of the format `gs://osv-vulnerabilities/<ECOSYSTEM>/<ID>.json`
+as well as a zip containing all vulnerabilities for each ecosystem at
+`gs://osv-vulnerabilities/<ECOSYSTEM>/all.zip`
```
Comment (inferno-chromium) [inline README.md:36]: maybe give gsutil command example to copy one db locally.

**Review by inferno-chromium — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (oliverchang) [reply README.md]: Done. Also linked to blog post. PTAL!

File: README.md (hunk 2 above)
Comment (oliverchang) [reply README.md:47]: done!

**Review by inferno-chromium — APPROVED** [review]

**Pull request merged**